src/components/__tests__/__snapshots__/ProgramList.test.js.snap

```diff
@@ -2,39 +2,23 @@
 
 exports[`The ProgramList component renders on the page with no errors 1`] = `
 <Fragment>
-  <Button
-    as={
-      Object {
-        "$$typeof": Symbol(react.forward_ref),
-        "displayName": "Link",
-        "propTypes": Object {
-          "innerRef": [Function],
-          "onClick": [Function],
-          "replace": [Function],
-          "target": [Function],
-          "to": [Function],
-        },
-        "render": [Function],
-      }
-    }
-    primary={true}
+  <WithStyles(ForwardRef(Button))
```

Comment:
This happened because the snapshot test needs another `dive()` since a new layer `WithStyles` was added

Reply:
This actually looks correct. It's just a difference in how the Material UI button is done